schema/collectionPoint.js

```diff
@@ -0,0 +1,20 @@
+const collectionPointSchema = `
+
+extend type Query {
+
+    collectionPoint(id: Int!): collectionPoint
+    collectionPoints: [collectionPoint]
+
+}
+type collectionPoint {
+
```

Comment:
there should be an `id` field in the schema

Reply:
I forgot to add that, I'll put the id in on my next push.